package.json

```diff
@@ -1,42 +1,8 @@
 {
   "name": "frame",
-  "version": "0.5.0-beta.18",
+  "version": "0.5.0-beta.19",
```

Comment:
make sure to base the branches off of beta.19, I think this change is already made there

Reply:
I just checked and I did actually base this off beta.19 🤷